phpstan.neon

```diff
@@ -0,0 +1,20 @@
+parameters:
+    ignoreErrors:
+        - '#Constant PHPUNIT_COMPOSER_INSTALL not found.#'
+        - '#Constant __PHPUNIT_PHAR__ not found.#'
+        - '#Constant __PHPUNIT_PHAR_ROOT__ not found.#'
+        - '#Variable \$e might not be defined.#'
+        - '#Variable \$method might not be defined.#'
+        - '#Variable \$_timeout might not be defined.#'
+        - '#Class SebastianBergmann\\Invoker\\Invoker not found.#'
+        - '#Instantiated class SebastianBergmann\\Invoker\\Invoker not found.#'
+        - '#Caught class SebastianBergmann\\Invoker\\TimeoutException not found.#'
+        - '#Variable \$filename might not be defined.#'
+        - '#Variable \$hasMetRequirements might not be defined.#'
+        - '#Variable \$testResult might not be defined.#'
+        - '#Variable \$color might not be defined.#'
+        - '#Variable \$stdout might not be defined.#'
+        - '#Variable \$stderr might not be defined.#'
+        - '#Variable \$className might not be defined.#'
+        - '#Variable \$testMethod might not be defined.#'
+        - '#Variable \$codeCoverage in isset\(\) always exists and is not nullable.#'
```

Comment:
Can you please report these `might not be defined` as issues to PHPStan? Why do you think they are not errors worth fixing?

Reply:
I think he's just honoring Sebastian's request:

> What I meant was that I need minimal, self-contained pull requests. One for the phpstan build stage and one for each of the code changes.

So they will tackle errors later.

Reply:
OK, I recommend starting with level 0 instead.

Reply:
Rather than ignoring something and then forgetting to fix that.

Reply:
Errors about `might not be defined` variables are starting with level 1, so if you're ignoring all those, you can start with level 0 😊

Reply:
Well i didn't knew that there was a level 0. But if @sebastianbergmann agree with you I can easily move the PR to that solution.